samples/templates/express-embedded-auth-with-sdk/web-server/routes/authenticator.js

```diff
@@ -273,4 +273,48 @@ router.post('/challenge-authenticator/google_otp', async (req, res, next) => {
   handleTransaction({ req, res, next, authClient, transaction });
 });
 
+// Handle Security Question authenticator
+router.get('/enroll-authenticator/security_question', async (req, res) => {
+  renderPage({
+    req, res,
+    render: () => renderTemplate(req, res, 'authenticator-security_question', {
+      req, res,
+      title: 'Enroll security question authenticator',
+      action: '/enroll-authenticator/security_question',
+      input: {
```

Comment:
Looks like `input ` is not correct here (should be array of `questionCode`, `question`, `answer`), right?
And it's not used in the template?

Reply:
questionCode and question are handled in the mustache template.